backend/cmd/headlamp.go

```diff
@@ -365,21 +365,33 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 	fmt.Println("*** Headlamp Server ***")
 	fmt.Println("  API Routers:")
 
-	if !config.useInCluster {
-		// load kubeConfig clusters
-		err := kubeconfig.LoadAndStoreKubeConfigs(config.kubeConfigStore, kubeConfigPath, kubeconfig.KubeConfig)
-		if err != nil {
+	// load kubeConfig clusters
+	err := kubeconfig.LoadAndStoreKubeConfigs(config.kubeConfigStore, kubeConfigPath, kubeconfig.KubeConfig)
+	if err != nil {
+		if config.useInCluster {
+			logger.Log(logger.LevelInfo, nil, err, "loading kubeconfig")
```

Comment:
maybe we could define the log into a variable and then only have one logger.Log call? I guess the code would be more easily understood.